src/pages/Home/components/Portfolio/Prompt/Prompt.tsx

```diff
@@ -659,20 +662,147 @@ export function Prompt() {
                 {prompts.map((message, i) => {
                   if (message.role === 'model') {
                     return (
-                      <AIDialog
-                        message={message}
-                        key={i}
-                        scrollToBottom={scrollToBottom}
-                        isDialogOpen={isDialogOpen}
-                      />
+                      <>
+                        <AIDialog
+                          message={message}
+                          key={i}
+                          scrollToBottom={scrollToBottom}
+                          isDialogOpen={isDialogOpen}
+                        />
+                        <Stack sx={{ flexDirection: 'row' }}>
+                          <Button
+                            variant="outlined"
+                            sx={{ my: 0.5, width: 'auto' }}
+                            onClick={() => {
+                              prompt('Switch Network');
+                            }}
+                          >
+                            Switch Network
+                          </Button>
+                          <Button
+                            variant="outlined"
+                            sx={{ my: 0.5, width: 'auto' }}
+                            onClick={() => {
+                              prompt('Tokens');
+                            }}
+                          >
+                            Tokens
+                          </Button>
+                        </Stack>
+                      </>
                     );
                   }
                   return <UserDialog message={message} key={i} />;
                 })}
                 {isTyping && <TypingAvatar />}
               </Stack>
             </Scrollbars>
-            <Stack sx={{ p: 2 }}>
+            <Stack
+              sx={{
+                p: 2,
+                flexDirection: 'row',
+                alignItems: 'center',
+                position: 'relative',
+              }}
+            >
+              <Slide in={isCommandsOpen} direction="right">
+                <Stack
+                  sx={{
+                    position: 'absolute',
+                    bottom: '80px',
+                    left: 0,
+                    width: 'auto',
+                    zIndex: 1,
+                  }}
+                >
+                  <Button
+                    variant="contained"
+                    sx={{ my: 0.5 }}
+                    onClick={() => {
+                      prompt('Tokens list');
+                      setIsCommandsOpen(false);
+                    }}
+                  >
```

Comment:
Please consider to use `gap` property of the `Stack` component to control the space between the buttons.
It will also remove all `sx={{ my: 0.5 }}` lines 
```suggestion
                <Stack
                 gap={0.5}
                  sx={{
                    position: 'absolute',
                    bottom: '80px',
                    left: 0,
                    width: 'auto',
                    zIndex: 1,
                  }}
                >
                  <Button
                    variant="contained"
                    onClick={() => {
                      prompt('Tokens list');
                      setIsCommandsOpen(false);
                    }}
                  >
```

Reply:
thank you but this was only a PoC PR I won't merge it anyway :)